src/bp/nlu/stan/api.ts

```diff
@@ -190,9 +192,9 @@ export default async function(options: APIOptions, engine: NLUEngine.Engine) {
       }
 
       const modelId = NLUEngine.modelIdService.fromString(stringId)
-      // once the model is loaded, there's no more password check
+      // TODO: once the model is loaded, there's no more check to appSecret and appId
```

Comment:
is this bad ?

Reply:
yes kind of: one could make predictions on a model without having the correct secret. This should really be fixed, but I can't think of anything not ugly